.gitignore

```diff
@@ -28,6 +28,7 @@ TAGS
 *.swo
 *.swp
 .gdb_history
+/.vscode/
 
```

Comment:
This doesn't look like it belongs in a commit with this message?

Reply:
It just came up while I was editing. I use VS Code to edit C files, and it created that directory which should be ignored by git and not included it the commit.

Reply:
I agree that directory should not be included in the commit but it isn't and I'm not talking about that. I'm saying that if you want to add that to the .gitignore file, it would be better to do it in a separate commit.

(This isn't a bid deal.)

Reply:
Ah okay, I can separate them.